assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -75,6 +75,27 @@ export default function PropertySelect( {
 		select( MODULES_ANALYTICS_4 ).getPropertyID()
 	);
 
+	const measurementIDs = useSelect( ( select ) => {
+		if ( ! properties?.length ) {
+			return null;
+		}
+
+		return select(
+			MODULES_ANALYTICS_4
+		).getMatchedMeasurementIDsByPropertyIDs( properties );
+	} );
+
+	const areMeasurementIDsResolving = useSelect( ( select ) => {
+		if ( ! properties?.length ) {
```

Comment:
If `properties` is `undefined` (still resolving) then this should probably be `true`. Only if `properties.length === 0` we could return early with `false`.